Documentation notifications-sms-api (#1405)
Altinn/altinn-studio-docs: content/notifications/architecture/sms/_index.en.md

weight: 2
toc: false
---
## API

### Public API

> The Altinn Sms Notifications API is an HTTP-based RESTful API that provides an endpoint for receiving delivery reports from the SMS provider. 

Reviewer: ```suggestion
```

This is also mentioned in relation to the concrete controller

- [DeliveryReportController](https://github.com/Altinn/altinn-notifications-sms/blob/main/src/Altinn.Notifications.Sms/Controllers/DeliveryReportController.cs)

>   Endpoint receiving delivery reports in XML-format.

Reviewer: ```suggestion
  Endpoint receiving delivery reports in XML-format from SMS provider.
```

weight: 2
toc: false
---
## API

### Public API
The Altinn Sms Notifications API is an HTTP-based RESTful API that provides an endpoint for receiving delivery reports from the SMS provider. 

> Link Mobility push deliveryreports for every sms sent through their gateway.

Reviewer: ```suggestion
```

weight: 2
toc: false
---
## API

### Public API

> ### Altinn Notifications Sms API

Reviewer: ```suggestion
```

Blir vel litt smør på flesk. Vi er inne i SMS seksjonen og overskriften over er API

Author: copy paste :p

> The Altinn Sms Notifications API is an HTTP-based RESTful API that provides an endpoint for receiving deliveryreports. 

Reviewer: ```suggestion
The Altinn Sms Notifications API is an HTTP-based RESTful API that provides an endpoint for receiving delivery reports. 
```

> The Altinn Sms Notifications API is an HTTP-based RESTful API that provides an endpoint for receiving delivery reports. 

Reviewer: ```suggestion
The Altinn Sms Notifications API is an HTTP-based RESTful API that provides an endpoint for receiving delivery reports from the SMS provider. 
```

Reviewer: Could we simplify it even further? 
All details are not required, we could add the suggestion below, and then move the information about link pushing to the integrations section where we cover link? 

### Public API 
 - [DeliveryReportController](https://github.com/Altinn/altinn-notifications-sms/blob/main/src/Altinn.Notifications.Sms/Controllers/DeliveryReportController.cs)
  Endpoint receiving delivery reports in XML-format from SMS provider. The controller is protected with [basic authentication](https://github.com/Altinn/altinn-notifications-sms/blob/main/src/Altinn.Notifications.Sms/Configuration/BasicAuthenticationHandler.cs).

## Run on local machine
Instructions on how to set up the service on local machine for development or testing is covered by 

> [the README in the repository](https://github.com/Altinn/altinn-notifications-email). 

Reviewer: ```suggestion
[the README in the repository](https://github.com/Altinn/altinn-notifications-sms). 
```
